.github/workflows/tnf-image.yaml

```diff
@@ -126,10 +126,33 @@ jobs:
           repository: test-network-function/cnf-certification-test-partner
           path: cnf-certification-test-partner
           ref: ${{ env.PARTNER_VERSION }}
-          
-      - name: Start the minikube cluster for `local-test-infra`
+
+      # Only one of the following 2 steps will work depending on the repo version
+
+      # For versions > 3.3.0
```

Comment:
this part here should work for older code as well ? did you tried it and it did not work ?

Reply:
yes. It works because it is checking the presence of the file first